webrtc.html

```diff
@@ -8543,6 +8543,30 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     </section>
     <section>
       <h4><dfn>RTCDTMFSender</dfn></h4>
+      <p>To create an <code>RTCDTMFSender</code>, the user agent MUST
+        run the following steps:</p>
+        <ol>
+          <li>
+            <p>Let <var>dtmf</var> be a newly created
+            <code><a>RTCDTMFSender</a></code> object.</p>
+          </li>
+          <li>
+            <p>Let <var>dtmf</var> have a <dfn>[[\CanInsertDtmf]]</dfn>
+            internal slot, initialized to <code>false</code>.</p>
+          </li>
+          <li>
+            <p>Let <var>dtmf</var> have a <dfn>[[\Duration]]</dfn>
+            internal slot.</p>
+          </li>
+          <li>
+            <p>Let <var>dtmf</var> have a <dfn>[[\InterToneGap]]</dfn>
+            internal slot.</p>
+          </li>
+          <li>
+            <p>Let <var>dtmf</var> have a <dfn>[[\ToneBuffer]]</dfn>
+            internal slot.</p>
```

Comment:
Where is [[ToneBuffer]] actually referenced?

Reply:
Added references.